code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -5,6 +5,10 @@
 
 	if(reagents)
 
+		if(locked_to && istype(locked_to,/obj/structure/bed/nest))//alien nest will keep their victims alive for a bit longer.
```

Comment:
I was under the impression that adding a Life/process tick to nests would be even worse but I guess I didn't think at the time that I could just have the proc stop when nobody is stuck on the nest.